crypto/fipsmodule/CMakeLists.txt

```diff
@@ -305,7 +305,7 @@ function(cpreprocess dest src)
   string(REGEX REPLACE "[ ]+" ";" CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS}")
   add_custom_command(
     OUTPUT ${dest}
-    COMMAND ${CMAKE_ASM_COMPILER} ${TARGET} ${CMAKE_ASM_FLAGS} -E ${src} ${PREFIX_INCLUDE} -I${PROJECT_SOURCE_DIR}/include > ${dest}
+    COMMAND ${CMAKE_ASM_COMPILER} ${TARGET} ${CMAKE_ASM_FLAGS} -E -dI ${src} ${PREFIX_INCLUDE} -I${PROJECT_SOURCE_DIR}/include > ${dest}
```

Comment:
My understanding is that the solution was to keep the #include directives using this preprocessor compilation flag, correct?
https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html
> -dI
    Output ‘#include’ directives in addition to the result of preprocessing.

Reply:
This comment isn't necessarily relevant to the actual fix, but is related to the solution. The real solution was to have the delocator inject in `#include` C preprocesser directives so that we can do prefixing after the delocator has executed.